src/components/Tooltip/Tooltip.react.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+import { Manager, Placement, Reference, Popper } from "react-popper";
+import type { PopperChildrenProps, ReferenceChildrenProps } from "react-popper";
+import styles from "./tooltip.css";
+
+type Props = {|
+  +content: string,
+  +children?: React.Node,
+  +className?: string,
+  +placement?: Placement,
+|};
+
+type State = {
+  isShown: boolean,
+};
+
+class Tooltip extends React.Component<Props, State> {
+  state = { isShown: false };
+
+  _handleTriggerOnMouseEnter = (e: SyntheticMouseEvent<HTMLElement>) => {
+    e.preventDefault();
+    this.setState({ isShown: true });
+  };
+
+  _handleTriggerOnMouseLeave = (e: SyntheticMouseEvent<HTMLElement>) => {
+    e.preventDefault();
+    this.setState({ isShown: false });
+  };
+
+  render(): React.Node {
+    const { className, children, placement, content } = this.props;
+
+    const classes = cn(
+      "tooltip",
+      placement ? "bs-tooltip-" + placement : null,
```

Comment:
Not sure, but you may be able to just do `placement && "bs-tooltip-" + placement,`